lib/util.js

```diff
@@ -67,6 +73,11 @@ module.exports.kickoff = function kickoff() {
           console.log(results);
           /* eslint-enable */
         }
+        done();
       }.bind(this));
+    }.bind(this);
```

Comment:
I'd much prefer that generally, we don't transpile right now and not sure if [we care to support 0.12 or 0.10 (some others)](http://node.green/#ES2015-functions-arrow-functions).

Would prefer adding better es6 support in another PR.